manuscript/teaching-reproducibility.Rmd

```diff
@@ -729,19 +729,39 @@ Additionally, we interleave these practices as
 grade while completing assignments focused on other data science topics.
 This latter strategy parallels how they will employ reproducibility 
 habits later in their carers
-(a detailed example follows in the next section).
-The tools we chose to teach with are data science reproducibility software
-that students are likely to employ in workflows in their future work.
-This sustained practice not only enforces students' habits,
-but also increases their proficiency using authentic reproducibility software
-and when they into problems, it is in an environment
-where they can easily reach out for help without feeling intimidated to ask.
+(a detailed example follows in the next section),
+and the skill bar is put at a level
+that we require all student to meet
+after introduced to version control topic
+during the first few weeks of class.
```

Comment:
This should be it's own sentence. And it needs to be edited. I find it hard to parse and digest. Or we could remove it?